INSTALL.md

```diff
@@ -99,35 +100,28 @@ thus does not need an include path.
 
 There are several BLAS implementations, depending on the OS and
 machine. To have reasonable performance, the BLAS library should be
-multithreaded.
-
-To check that the link flags are correct, and verify whether the
-implementation uses 32 or 64 bit integers, you can
-
-  `make test_blas`
-
-and run
-
-  `./misc/test_blas`
+multithreaded. On Intel platforms, MKL is the fastest BLAS implementation, 
+see flags above on how to enable it.
 
 
 Testing Faiss
 -------------
 
 A basic usage example is in
 
-  `demos/demo_ivfpq_indexing`
+  `demos/demo_ivfpq_indexing.cpp`
 
-which you can build by calling
-  `make demo_ivfpq_indexing`
+which you can build by calling 
+
+** how to run cmake in the demos subdirectory **
 
 It makes a small index, stores it and performs some searches. A normal
 runtime is around 20s. With a fast machine and Intel MKL's BLAS it
 runs in 2.5s.
 
 To run the whole test suite:
 
-   `make test` (for the CPU part)
+** how to run the C++ tests **
```

Comment:
`$ make -C build test`

Reply:
Does not work
```
(faiss_latest_cmake) matthijs@devfair0144:~/faiss_versions/faiss_latest_cmake$ make -C build test
make: Entering directory '/private/home/matthijs/faiss_versions/faiss_latest_cmake/build'
make: *** No rule to make target 'test'.  Stop.
make: Leaving directory '/private/home/matthijs/faiss_versions/faiss_latest_cmake/build'
(faiss_latest_cmake) matthijs@devfair0144:~/faiss_versions/faiss_latest_cmake$ make -C build tests
make: Entering directory '/private/home/matthijs/faiss_versions/faiss_latest_cmake/build'
make: *** No rule to make target 'tests'.  Stop.
make: Leaving directory '/private/home/matthijs/faiss_versions/faiss_latest_cmake/build'
```

Reply:
In order to build the tests, they need to be enabled via `-DBUILD_TESTING=ON`. The option should be `ON` by default, but it is currently not, I'll send a fix.